additionalCSS/aboutMe.css

```diff
@@ -0,0 +1,88 @@
+.aboutMe {
+  width: 100%;
+  align-items: center;
+}
```

Comment:
- Kindly make sure the content of the About Section is aligned using flexbox according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=Make%20sure%20the%20content%20of%20the%20%22About%20myself%22%20section%20is%20aligned%20with%20flexbox.)

Reply:
@GedeonTS Can you please go in details on what you mean by this?